tests/scalability/package.json

```diff
@@ -5,8 +5,8 @@
   "dependencies": {
     "csv-parser": "^3.0.0",
     "minimist": "^1.2.7",
-    "pouchdb": "^7.3.1",
-    "pouchdb-adapter-memory": "^7.3.1",
```

Comment:
Would it be better to remove dependencies that are already available in the root? It makes it much easier to manage bumping dependencies...

Reply:
Agree!